src/interface.rs

```diff
@@ -224,8 +224,9 @@ impl HWIClient {
         path: &DerivationPath,
         expert: bool,
     ) -> Result<HWIExtendedPubKey, Error> {
+        let prefixed_path = format!("m/{}", path);
```

Comment:
Drats, that turned out ugly didn't it. Maybe we should have a function that explicitly does this.

Reply:
Yes, a specific function for that would help.

Reply:
Including `m/` is actually incorrect/misunderstanding of the BIP. Why did you keep the incorrect value?

Reply:
Hey @Kixunil , which BIP in particular do you mean? 

Anyway this PR isn't meant to change how this lib works, only to bump the version of rust-bitcoin. If there's something we need to fix we can do it in another PR.

Reply:
32, see this issue: https://github.com/rust-bitcoin/rust-bitcoin/pull/2451

However, I've just noticed this is still debated: https://github.com/rust-bitcoin/rust-bitcoin/issues/2671

Reply:
> Including `m/` is actually incorrect/misunderstanding of the BIP. Why did you keep the incorrect behavior?

> Hey @Kixunil , which BIP in particular do you mean?
> 
> Anyway this PR isn't meant to change how this lib works, only to bump the version of rust-bitcoin. If there's something we need to fix we can do it in another PR.

Yes, I kept it because it would require changes on the upstream [hwilib](https://github.com/bitcoin-core/HWI), ACK's, and so on, as Steve mentioned not the scope of this PR.

Thanks, I'll keep an eye on the issue discussion, and update it in another PR if appropriate.